rs/sns/swap/src/swap.rs

```diff
@@ -3472,6 +3362,144 @@ fn create_sns_neuron_basket_for_neurons_fund_participant(
     Ok(recipes)
 }
 
+enum ConversionError {
+    Invalid,
+    Skip,
```

Comment:
The name of this type isIMHO not verbose enough to convey what situation has been labeled by instances of this type.